examples/esp_idf/stream/pytest/test_sample.py

```diff
@@ -0,0 +1,36 @@
+# Copyright (c) 2024 Golioth, Inc.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import contextlib
+import logging
+import pytest
+import time
+import yaml
+import datetime
+import re
+
+LOGGER = logging.getLogger(__name__)
+
+pytestmark = pytest.mark.anyio
+
+async def test_stream(board, device):
+    time.sleep(2)
+
+    # Set Golioth credential
+    golioth_cred = (await device.credentials.list())[0]
+    board.set_golioth_psk_credentials(golioth_cred.identity, golioth_cred.key)
+
+    # Wait for device to connect
+    board.wait_for_regex_in_line('.*Golioth client connected', timeout_s=30.0)
+
+    # Verify temp messages
+
+    lower_temp = 20.0
+    upper_temp = 21.5
+    async with contextlib.aclosing(device.stream.iter()) as stream_iter:
+        async for value in stream_iter:
+            LOGGER.info("ts: {0}, temp: {1}".format(value['timestamp'], value['data']['sensor']['temp']))
+            assert (value["data"]["sensor"]["temp"] >= lower_temp and value["data"]["sensor"]["temp"] <= upper_temp)
+            if value["data"]["sensor"]["temp"] == upper_temp:
+                break
```

Comment:
In b85446ba73a4 ("examples: zephyr: stream: use CBOR over JSON") we removed `sensor` from the path where data goes. Should we remove it here as well to keep samples consistent for all platforms?